src/store/dashboard/ociDashboard/ociDashboardWidgets.ts

```diff
@@ -40,7 +40,7 @@ export const costSummaryWidget: OciDashboardWidget = {
     titleKey: messages.ociCostTrendTitle,
     type: ChartType.rolling,
   },
-  availableTabs: [OciDashboardTab.service_names, OciDashboardTab.payer_tenant_ids, OciDashboardTab.resource_locations],
+  availableTabs: [OciDashboardTab.service_names, OciDashboardTab.payer_tenant_ids, OciDashboardTab.resources],
```

Comment:
I believe `OciDashboardTab.service_names` should be `OciDashboardTab. product_services`? And `OciDashboardTab.resources` should be `OciDashboardTab.regions`